### PR TITLE
using makefile instead of tasks

### PR DESCRIPTION
**Pull request by zoli**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	@go fmt ./lib/...
+license:
+	@go run license.go lib
+
+checkfmt:
+ifneq ($(shell gofmt -l lib),)
+	$(error code not fmted, run make fmt. $(shell gofmt -l lib))
+endif
+
+checklisence:
```
Comment (erbridge) [inline Makefile]: Typo here. Should be `checklicense`.

Comment (erbridge) [inline Makefile]: Or even `check_license` to be consistent with the style of other tasks. Same with `check_fmt`.

**Comment by zoli** [comment]
I also suggest 2 more changes on https://github.com/limetext/lime-backend

- IMHO the name of the repo should be limetext/backend not limtext/lime-backend

- Also we can now remove the lib directory and move the contents to repo top directory
 although I'm not sure this is a good idea

resolves #94

**Comment by erbridge** [comment]
Agree about renaming `lime-backend` to `backend`. Also if the need for the `tasks` folder is gone, I don't see why not to move `lib` into the root.

**Comment by erbridge** [comment]
Though I'm not sure I'm a fan of putting `gen_license.go` in the root...

**Comment by zoli** [comment]
> Though I'm not sure I'm a fan of putting gen_license.go in the root...

I don't think this is good to also as you said its duplicated across all the projects. I will work on a better solution for this later.

**Pull request merged**